### DISCUSSION TITLE
LAB 02.03 - Pandas-Task 1: Extract data

### DISCUSSION THREAD

**Discussion by rojo32** [rramosp/ai4eng.v1#24] [02 - PYTHON]
Para el siguiente ejercicio.

![image](https://user-images.githubusercontent.com/5897347/121823793-12922000-cc6d-11eb-8dbe-bbe632451173.png)

presento el siguiente error

![image](https://user-images.githubusercontent.com/5897347/121823823-40776480-cc6d-11eb-9673-2dd6affcb715.png)

![image](https://user-images.githubusercontent.com/5897347/121823832-54bb6180-cc6d-11eb-8ec9-98afe98336e4.png)

**Comment by DanielMTU** [comment]
Hola, 
Como se indica en el enunciado del ejercicio, **la columna 'margin' puede estar presente  o no**, es por eso que al momento de que se realiza la evaluación, en los casos donde no aparece la columna 'margin' resulta en error.

**Reply by rramosp** [reply]
gracias rojo32 por la pregunta y DanielMTU por la respuesta
